docs/jetstream/jetstream.md

```diff
@@ -57,7 +57,7 @@ for which all clients have complete data.
 
 When analyzing experiments, the following steps are executed for each experiment:
 
-![Experiment analyis steps](https://github.com/mozilla/experimenter-docs/blob/main/static/img/jetstream/analysis-steps.png)
+<img src="/img/jetstream/analysis-steps.png" alt="Experiment analyis steps" width="350"/>
```

Comment:
I'm not opposed to `<img />` tags but I'm curious if Markdown image syntax straight up doesn't work? Or something else?

Also will want to check if that explicit 350px width works on all screen sizes

Reply:
The reason for `<img>` is to scale the image down. Looking at some other images in the docs, it's probably better to use `className="img-sm"` instead though.